MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -232,10 +232,19 @@ MantidUI::MantidUI(ApplicationWindow *aw)
     qRegisterMetaType<std::string>();
   }
 
-  m_exploreMantid = boost::make_shared<QWorkspaceDockView>(this, aw);
+  m_exploreMantid = boost::make_shared<QWorkspaceWidget>(this);
```

Comment:
I don't think we need to manage this with a `shared_ptr`. None of the other widgets here do that. The Qt parent/child relationship should handle the clean up.